### PR TITLE
Update CLI readme

### PR DESCRIPTION
**Pull request by EsterASimoes**
- Include explanations of project id, collection id and dataset id and how to retrieve them to be used in different cli commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ Create a collection using a combined dataset and imageset config.
 ```
 zeg create collections --project [Project Id] --token [API token] --url [Server url] --config [path to configuration yaml]
 ```
+
+In here, project id is a synonym for workspace id, and you'll be able to find it in the url when looking at a given collection workspace:
```
Comment (tarqu1n) [inline README.md:37]: Generally i'd like to avoid personal point of view perspective writing in documentation. Content is fine though. Maybe reword to:

```
Project id, or workspace id, can be found in the url of a collection or collection listing page. For example:

https://zegami.com/mycollections/66xtfqsk

In the case of this workspace, it's `66xtfqsk`.
```

File: README.md [hunk 2]
```diff
@@ -114,6 +121,11 @@ zeg delete collections [collection id] --project [Project Id] --token [API token
 ```
 zeg publish collection [collection id] --project [Project Id] --config [path to configuration yaml] --token [API token]
 ```
+Similarly to the workspace id, the collection id can be found in the url pointing to a given collection. For instance:
```
Comment (tarqu1n) [inline README.md:124]: ```Similarly to the workspace id, the collection id can be found in the url **for** a given collection. For instance:```

File: README.md [hunk 3]
```diff
@@ -132,6 +144,11 @@ Get a data set
 ```
 zeg get dataset [dataset id] --project [Project Id] --token [API token]
 ```
+The dataset id can be found by by running the following on the cli:
```
Comment (tarqu1n) [inline README.md:147]: `Dataset Ids can be found in the collection information, obtained by running:`

File: README.md [hunk 4]
```diff
@@ -132,6 +144,11 @@ Get a data set
...
+
+zeg get collection <collection id> --project <project id>
+
+From here we will be able to see the upload_dataset_id which can be thought of as the data input funnel (whereas dataset_id is the data output pipe that the viewer pulls from).
```
Comment (tarqu1n) [inline README.md:151]: ```From here `upload_dataset_id` can be obtained. This identifies the dataset that represents the data as it was uploaded. Whereas `dataset_id`  identifies the dataset processed delivered to the viewer.```

Comment (EsterASimoes) [reply README.md:151]: thanks for the feedback @tarqu1n ! Will implement now 👍
